packages/react-recommendations/README.md

```diff
@@ -176,17 +252,93 @@ The product component to display.
 > `Translations`
 
 ```ts
-type Translations = {
+type RecommendationTranslations = Partial<{
   title: string;
-};
+  showMore: string;
+}>;
 ```
 
 ##### `children`
 
-> `(props) => JSX.Element`
+> `(props: ChildrenProps) => JSX.Element`
+
+<details>
+
+<summary><code>ChildrenProps</code></summary>
+
+```ts
+type ChildrenProps<TObject> = {
+  recommendations: TObject[];
+  View(props: unknown): JSX.Element;
+  translations: RecommendationTranslations;
+};
+```
+
+</details>
 
 Render function to modify the default rendering.
 
+The default implementation is:
+
+```js
+function defaultRender(props) {
+  if (props.recommendations.length === 0) {
+    return null;
+  }
+
+  return (
+    <section className="auc-Recommendations">
+      {props.translations.title && <h3>{props.translations.title}</h3>}
+
+      <props.View />
+    </section>
+  );
+}
+```
+
+##### `view`
+
+> `(props: ViewProps) => JSX.Element`
+
+<details>
+
+<summary><code>ViewProps</code></summary>
+
+```ts
+type ViewProps<
+  TItem extends RecordWithObjectID,
+  TTranslations extends Record<string, string>,
+  TClassNames extends Record<string, string>
+> = {
+  items: TItem[];
+  itemComponent({ item: TItem }): JSX.Element;
+  classNames?: Partial<TClassNames>;
```

Comment:
We don't pass `classNames`.

Reply:
Make sense

Reply:
Done in https://github.com/algolia/ui-components/pull/18/commits/25fa8410a96e0521a21ab5738b56d938434a2622